miio/viomivacuum.py

```diff
@@ -201,17 +268,24 @@ def is_on(self) -> bool:
         return self.state in cleaning_states
 
     @property
-    def mode(self):
-        """Active mode.
+    def edge_state(self) -> ViomiEdgeState:
+        """Vaccum along the edges
 
-        TODO: is this same as mop_type property?
+        The settings is valid once
+        0: disabled
+        2: enabled
+        5: unknown
         """
-        return ViomiMode(self.data["mode"])
+        return ViomiEdgeState(self.data["mode"])
 
     @property
-    def mop_type(self):
-        """Unknown mop_type values."""
-        return self.data["mop_type"]
+    def mop_installed(self) -> bool:
+        """Mop installed status
+
+        True if the mop is installed
+        False if the mop is NOT installed
```

Comment:
```suggestion
        """True if mop is installed.
```
This delivers the same information with much shorter docstring :-) Less is more sometimes!

Reply:
fixed